app/services/audit/event/util.rb

```diff
@@ -0,0 +1,18 @@
+module Audit
+  module Event
+    class Util
+      class << self
+        def deserialize(string)
+          ActiveSupport::JSON.decode(string).deep_symbolize_keys.tap do |h|
+            h[:time] = Time.iso8601(h[:time])
+            h[:end] = Time.iso8601(h[:end])
```

Comment:
This should be Time.zone.iso8601 or Time.zone.parse if there's any chance the time string is not ISO

Reply:
Please see https://github.com/thepracticaldev/dev.to/pull/3449#discussion_r304240866